org.kde.sweeper.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id": "org.kde.sweeper",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "rename-icon": "sweeper",
+    "command": "sweeper",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.kde.sweeper",
+        "--filesystem=xdg-cache"
```

Comment:
why would it have this permission? it's the build failure.

Reply:
It needs access to `~/.cache`.